driver-core/src/main/java/com/datastax/driver/core/RegularStatement.java

```diff
@@ -95,6 +99,30 @@ public String getQueryString() {
     public abstract ByteBuffer[] getValues(ProtocolVersion protocolVersion, CodecRegistry codecRegistry);
 
     /**
+     * The named values to use for this statement.
+     * <p/>
+     * A statement can use either positional or named values, but not both. So if this method returns a non-null result,
+     * {@link #getValues(ProtocolVersion, CodecRegistry)} will return {@code null}.
+     * <p/>
+     * Values for a RegularStatement (i.e. if this method does not return
+     * {@code null}) are not supported with the native protocol version 1: you
```

Comment:
Technically, the correct sentence would be "... if this method *or the following one* does not return null".